tests/FakeItEasy.Specs/CreationOptionsSpecs.cs

```diff
@@ -725,7 +725,7 @@ public interface IInterfaceThatWeWillAddAttributesTo3
                     var customAttributeBuilders1 = new List<CustomAttributeBuilder> { attribute1 };
 
                     var constructor2 = typeof(ExampleAttribute).GetConstructor(new[] { typeof(object[]) });
-                    var attribute2 = new CustomAttributeBuilder(constructor2, new[] { new object[] { 1, null } });
+                    var attribute2 = new CustomAttributeBuilder(constructor2, new object[] { new object[] { 1, null } });

```

Comment:
Well, it wasn't really an issue in that case, since the array is never written to after it's created, but it's a valid warning in general...